pkg/plugins/policies/core/rules/merge/merge.go

```diff
@@ -363,7 +363,12 @@ func appendSlices(dst reflect.Value, src reflect.Value) {
 			if dstField.Elem().Kind() == reflect.Slice {
 				mergeByKey := strings.Contains(field.Tag.Get(policyMergeTag), mergeValuesByKey)
 				if strings.HasPrefix(field.Name, appendSlicesPrefix) || mergeByKey {
-					s := reflect.AppendSlice(dstField.Elem(), srcField.Elem())
+					var s reflect.Value
+					if srcField.Elem().Kind() == reflect.Invalid {
+						s = reflect.AppendSlice(dstField.Elem(), reflect.Zero(dstField.Elem().Type()))
```

Comment:
you don't need to call AppendSlice when src is empty

Reply:
👍